src/app/hooks/useViewTracker/index.jsx

```diff
@@ -104,6 +104,10 @@ const useViewTracker = (props = {}) => {
             advertiserID,
             url,
             detailedPlacement,
+            ...(optimizelyVariation &&
+              optimizelyVariation !== 'off' && {
```

Comment:
Not sure about this `off` check. `off` could be a valid variation technically, so maybe we should allow that through as a variation in the ATI stats?